mcts/certsuite/cert.py

```diff
@@ -615,7 +621,7 @@ def get_application_ini(dm):
 def get_buildprop(dm):
     # get build properties
     buildprops = {}
-    buildpropoutput = dm.shellCheckOutput(["cat", "/system/build.prop"])
+    buildpropoutput = dm.shell_output("cat /system/build.prop")
```

Comment:
Hi Walter,

  I think this change is based on we use adbb2g instead of devicemanage, so it's the original method from adbb2g, we probalby not want to have another wrapper for this...